expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -65,12 +67,17 @@ public ResourceExpansionServiceImpl(ResourceService resourceService,
     }
 
     @Override
-    public ExpandedDataEntry expandEntry(Entity dataEntry) throws JsonProcessingException, NotFoundException {
+    public ExpandedDataEntry expandEntry(Entity dataEntry, boolean replaceContext) throws JsonProcessingException,
```

Comment:
Rename to `inlineUriContext` or `uriContext`? With `replaceContext`, I was asking the question: "To what?"

Reply:
What about shouldReplaceContext?